src/newSeed/stage5.html

```diff
@@ -8,7 +8,7 @@
       <div>
         <div class="infoInput">
           <label for="email">Contact Email</label>
-          <input id="email" type="email" placeholder="Valid email address"
+          <input id="email" type="email" placeholder="Please enter a valid email address"
```

Comment:
I don't agree with this change.  Placeholders are always rounds, not requests.  The wording you are proposing is appropriate for validation messages.

A better wording: "Project email address"

Reply:
Okay well @BartuCulha please note this also, thanks